setup.py

```diff
@@ -47,7 +48,7 @@
         'Intended Audience :: Developers',
         'Intended Audience :: System Administrators',
         'Natural Language :: English',
-        'License :: OSI Approved :: Apache Software License'
+        'License :: OSI Approved :: Apache Software License',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
```

Comment:
README says that Python 3.5 or newer is needed, so I think you need remove 3.3 and 3.4.

Reply:
Yeah, I overlooked it, thanks.